src/Umbraco.Web/Editors/AuthenticationController.cs

```diff
@@ -303,9 +303,27 @@ public UserDetail GetCurrentUser()
             var result = await UserManager.ResetPasswordAsync(model.UserId, model.ResetCode, model.Password);
             if (result.Succeeded)
             {
+                var lockedOut = await UserManager.IsLockedOutAsync(model.UserId);

```

Comment:
The property `LockoutEnabled` is incorrect. This property is for something that we don't support. In ASP.NET identity you can specify whether locking out a user is allowed on a per user basis - we do not support this so changing this flag does nothing.

Further to that you shouldn't manipulate those properties directly, the user manager has methods to do this, for example if you did want to modify this you'd use `UserManager.SetLockoutEnabled(true/false)` but in Umbraco this serves no purpose, see: https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/Security/BackOfficeUserStore.cs#L611 

A user is determined to be locked out purely by the date/time assigned to the lockout date: https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/Security/BackOfficeUserStore.cs#L542 

which you can set with the `UserManager.SetLockoutEndDateAsync` or `UserManager.SetLockoutEndDate`